arrays/arrays.c

```diff
@@ -19,45 +19,50 @@ typedef struct Array {
 /*****
  * Allocate memory for a new array
  *****/
-Array *create_array (int capacity) {
+Array *create_array(int capacity)
```

Comment:
I notice you don't return the array that you created in this function. Notice that the return type of this function is `Array *`, so you just need to add a `return array` line at the end of this function. Incidentally, without that line I was able to compile your code, but I couldn't run it. I got a seg fault. I would suspect that to be why you were having the issue with your test, but I saw you mentioned that the code still printed correctly. Whatever the case you should add that line and see if it enables you to pass the tests.